apps/graphql/src/apps/itinerary/Itinerary.js

```diff
@@ -32,7 +32,7 @@ export type ItinerariesOneWaySearchParameters = {|
   +sort?: Sort,
   +itinerary: {|
     +origin: SearchLocation,
-    +destination?: SearchLocation,
+    +destination: SearchLocation,
```

Comment:
Are we use this type `ItinerariesCommonSearchParameters` somewhere?

Reply:
Not anymore! will remove it

Reply:
Done!